README.md

```diff
@@ -28,31 +28,159 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## Authentication
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+Planet's APIs require an account for use.
+[Sign up here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+Authentication information can be obtained and stored locally with the CLI
+`planet init` command. Both the Python library and CLI will default to using
```

Comment:
This documented behavior of `planet init` is not synchronized with the actual behavior in this branch. What are thoughts about getting the behavior and the documentation to play nice?

Reply:
agree, this PR doesn't yet implement CLI behavior. @jreiberkyle let's remove all CLI-related documentation from the PR and we can put it back in down the road.